pipelines/py/bundle.yaml

```diff
@@ -0,0 +1,36 @@
+name: Install Python packages with UV
+
+needs:
+  packages:
+    - python-${{inputs.python-version}}-base
+    - uv
+
+inputs:
+  dependencies:
+    description: "Indirect dependencies to install."
```

Comment:
It's not clear to me what this means, or what syntax to use to specify them.

Reply:
We have some packages like jupyter base notebook that install indirect dependencies. This allows you to pass a list of indirect deps